docs/testing.md

```diff
@@ -17,3 +17,7 @@ You can use `npm run render $query $depth $aggregateMode` to output an HTML like
 The parameters are the same as the [`{{compat}}` macro](https://github.com/mdn/kumascript/blob/master/macros/Compat.ejs).
 
 Paste the generated HTML into the MDN editor (source mode). You can use a new page, for example: https://developer.mozilla.org/en-US/docs/new and verify if the output looks correct.
+
+## Statistics
+
+To see how changes will affect the statistics of real, true, and null values, you can run `npm run stats`.  This generates a Markdown-formatted table of the percentages of real, true, and null values for the eight primary browsers that browser-compat-data is focusing on.
```

Comment:
This is a great start, @vinyldarkscratch. Two things I'd like to see here: a definition of _real_ values and a link to #3555 (which is where that terminology originates, if I remember correctly).

Reply:
Sure thing!  Just added a line to define it -- how's it look?

Reply:
Suggested some nitpicky changes, but otherwise, looks great!

Reply:
💜

Suggestion applied!